### ISSUE TITLE
Update to webpack 2

### ISSUE THREAD

**Issue by rmoorman** [mikechau/sri-stats-webpack-plugin#71]
Hello,

I am currently using this plugin to output SRI information to a separate file. Sadly, the current version on npm does not support webpack 2.
However there is a [fork][fork] over at @sopherio which contains changes from @sechel that seem to work for webpack 2. Could those be merged and released to npm?

Thank you very much in advance!

Best regards,
Rico

[fork]: https://github.com/sopherio/sri-stats-webpack-plugin

**Comment by sechel** [comment]
Hey Rico,
why don't you just do this:
```json
"devDependencies": {
  "sri-stats-webpack-plugin": "github:sopherio/sri-stats-webpack-plugin"
}
```
It's not optimal in the sense that you do not have the full control over the version but in this case this seems not too bad. If you are worried, you could even fork ours and be safe.
Greetings
Stefan

**Comment by rmoorman** [comment]
@sechel thank you for the suggestion and I am actually already using this inside of my package.json :-)
~~~~
"sri-stats-webpack-plugin": "github:sopherio/sri-stats-webpack-plugin#f23de19e1d84dab50e59076e05bcafa508ba64c4"
~~~~
As for opening this issue here, it looked to me that this is the repo behind the version on npm which is what folks would normally use (upstream?) and I thought it could be a good thing to combine the efforts and keep the package up to date on npm :-)

**Comment by sechel** [comment]
You are right, it's always good to have it on npmjs.com. Thats where folks (and npm) search for things, and its even better since they decided to make organizations free :-) Big Up!

**Comment by ajsharp** [comment]
@mikechau Any plans to support this?

**Comment by mikechau** [comment]
Hi @ajsharp, yeah I will look into it this weekend. Currently the test is failing on custom-stats-patch: https://github.com/mikechau/webpack-custom-stats-patch/issues/49#issuecomment-272021612 and would like to resolve it before updating this package.

**Comment by mvastola** [comment]
Just a gentle ping to see if there has been (or is going to be) any movement on this.
